.github/mergeable.yml

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+version: 2
+mergeable:
+  - when: pull_request.*, pull_request_review.*
+    validate:
+      - do: title
+        # Do not merge when it is marked work in progress (WIP)
+        must_exclude:
+          regex: ^\[WIP\]
+          message: This is work in progress. Do not merge yet.
+        begins_with:
+          match: '[AIRFLOW-'
+        must_include:
+          regex: ^(\[AIRFLOW-XXXX\]|\[AIRFLOW\-\d{1,4}\])
```

Comment:
This should be `AIRFLOW-XXX` not `XXXX` :)

Reply:
:D This was intentional as there was a recent change in Github PR template that asked users to add 4 X's `[AIRFLOW-XXXX]` for docs.

Reply:
That reminds you that if you have XXX in the release scripts to distinguish doc-only updates, it should be updated to support also XXXX (maybe - depends if you have in search pattern "]")

Reply:
BTW. I really love that people will have to manually click the checkboxes now (though initially people won't understand why mergable complains). Happened already with @michalslowikowski00 :D. But I think this is good if they ask.